### DISCUSSION TITLE
New features ?

### DISCUSSION THREAD

**Discussion by Ruddle** [Ruddle/RemoteCam#3] [Ideas]
Please share your ideas here

**Comment by Heus-Sueh** [comment]
Is it possible to add microphone support?
There is a proprietary application that does this, but it uses pulseaudio and not pipewire to create a virtual audio device, and creating this virtual audio device is too complicated for a lay user.

Here is their documentation: https://docs.audiorelay.net/instructions/linux/use-your-phone-as-a-mic-for-a-linux-pc#create-a-permanent-virtual-audio-device-recommended

**Comment by Heus-Sueh** [comment]
Another suggestion would be a migration to [Material 3](https://m3.material.io/components) to make the app more attractive and beautiful

![image](https://github.com/Ruddle/RemoteCam/assets/98482469/0aaf6b7c-0dc8-415b-b38d-957f4fa1ffc8)

![image](https://github.com/Ruddle/RemoteCam/assets/98482469/b2b55de8-617e-48fe-ab89-0e64403338de)

**Comment by SpaceBallz2k8** [comment]
Any chance we could have a USB connection? App works great but at high resolutions and quality if the wifi quality drops the stream freezes up. But apart from that, which is not your app's fault its amazing.

**Reply by Ajaythepower1606** [reply]
Yeh need usb option....

**Comment by multigamesystem** [comment]
Exposure control at the very least would be really nice.  Like turn off auto exposure or set it at a set iso.
